File: Localization/ClientApp/src/app/models/Reports/TranslatedWordsReportRow.ts
export class TranslatedWordsReportRow {
  name_text?: string;
  language?: string;
  work_Type?: string;
  translations?: string;


  constructor(name: string, language: string, workType: string, translations: string) {
    this.name_text = name;
    this.language = language;
    this.translations = translations;
    this.work_Type = workType;
  }
}

File: Models/DatabaseEntities/Reports/TranslatedWordsReportRow.cs
﻿
using System;

namespace Models.DatabaseEntities.Reports
{
    /// <summary>
    /// Строка отчета по количеству переведенных строк/слов
    /// </summary>
    [Serializable]
    public class TranslatedWordsReportRow : BaseReportRow
    {
        /// <summary>
        /// Имя пользователя осуществлявшего перевод
        /// </summary>
        public string name_text { get; set; }

        /// <summary>
        /// Языки на которые переводил пользовател в указанный период времени
        /// </summary>
        public string language { get; set; }

        /// <summary>
        /// Вид работы
        /// </summary>
        public string work_Type { get; set; }

        /// <summary>
        /// Количество переводов
        /// </summary>
        public int translations { get; set; }
    }
}
